src/cmd/initContainer.go

```diff
@@ -172,6 +172,14 @@ func initContainer(cmd *cobra.Command, args []string) error {
 				}
 			}
 
+			if _, err := os.Readlink("/run/udev"); err != nil {
```

Comment:
It would be better to do it as a bind mount instead.

For individual regular files we prefer symbolic links instead of bind mounts, because symbolic links can track updates to the original file. Whereas, with bound mounts, if the original file was updated [atomically](http://www.microhowto.info/howto/atomically_rewrite_the_content_of_a_file.html), which is what most programs do, then the atomic rename changes the inode, and the two diverge.

However, symbolic links are always weird and introduce various oddities (eg., `stat` versus `lstat`), so bind mount is better when possible.

Reply:
I took the liberty to switch it to a bind mount.